services/web/client/source/class/osparc/dashboard/Dashboard.js

```diff
@@ -70,12 +70,14 @@ qx.Class.define("osparc.dashboard.Dashboard", {
     },
 
     __createMainViewLayout: function() {
-      [
+      const tabs = [
```

Comment:
I like this syntax for better readability:
```javascript
const tabs = [
  { label: 'label', buildLayout: () => {} }
  // ...
]
// iterate
tabs.forEach(({ label, buildLayout }) => {
  const tabPage = new qx.ui.tabview.Page(label)
  const layout = buildLayout()
})
```

Reply:
thanks 👌